docs/configure/typescript.md

```diff
@@ -6,34 +6,19 @@ Storybook provides an integrated [TypeScript](https://www.typescriptlang.org/) e
 
 ## Configure Storybook with TypeScript
 
-Storybook's configuration file (i.e., `main.ts`) is defined as an ESM module written in TypeScript, providing you with the baseline configuration to support your existing framework while enabling you stricter type-checking and autocompletion in your editor. Below is an abridged configuration file and additional information about the various configuration elements.
+Storybook's configuration file (i.e., `main.ts`) is defined as an ESM module written in TypeScript, providing you with the baseline configuration to support your existing framework while enabling you stricter type-checking and autocompletion in your editor. Below is an abridged configuration file.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/storybook-main-default-setup.ts.mdx',
+    'common/storybook-main-add-ts-config.ts.mdx',
   ]}
 />
 
-
 <!-- prettier-ignore-end -->
 
-| Configuration element | Description                                                                                                                                                                                              |
```

Comment:
This entire table is now better documented by the API reference. Not entirely sure why it was in the `configure/typescript` guide to begin with, honestly.

Reply:
That table was leftover from the updates to the TypeScript documentation for 7.0 as to give an overview of the options available, as cannot ascertain if the reader would check it on the overview page or not. But now that the API documentation is up, I'm good with removing it